src/components/Search.js

```diff
@@ -0,0 +1,13 @@
+import './Search.css';
+
+export default function Search(){
+    return(
+        <>
+        <div>
+        <input type="text" /> <br/>
+        <button>Search characters</button>
+        </div>
+        </>
+    );
```

Comment:
```suggestion
          <div>
            <input type="text" /> <br/>
            <button>Search characters</button>
          </div>
        </>
    );
```

indentationra figyelj